### DISCUSSION TITLE
Course Exercise

### DISCUSSION THREAD

**Discussion by 7gbice** [Cyfrin/security-and-auditing-full-course-s23#13] [Q&A]
# At the end of the Course Introduction Section, the Exercise given on
 ...
Section 0 NFT Challenge 👀
[Welcome! (Arb)](https://arbiscan.io/address/0xf923431da74ecc873c4d641fbdfa2564baafca9f#code)
[Welcome! (Sepolia)](https://sepolia.etherscan.io/address/0x39338138414df90ec67dc2ee046ab78bcd4f56d9)
...
What am i required of?

**Comment by akhilmanga** [comment]
Go to the write contract section and just write your twitter handle.

<img width="1372" alt="Screenshot 2023-12-11 at 4 14 48 PM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/116151859/c0336a23-edaa-41b5-a644-fb80660f2e25">

**Reply by cromewar** [reply]
Hello @akhilmanga 
This are supposed to be challenges but as it's the first one, just input for twitter handle 😉

**Reply by akhilmanga** [reply]
exactly
